eth/handler.go

```diff
@@ -864,18 +876,28 @@ func (pm *ProtocolManager) NodeInfo() *NodeInfo {
 	}
 }
 
-func (pm *ProtocolManager) FindPeers(targets map[common.Address]bool) map[common.Address]consensus.Peer {
-	m := make(map[common.Address]consensus.Peer)
+func (pm *ProtocolManager) FindPeers(targets map[enode.ID]bool) map[enode.ID]consensus.Peer {
+	m := make(map[enode.ID]consensus.Peer)
 	for _, p := range pm.peers.Peers() {
-		pubKey := p.Node().Pubkey()
-		addr := crypto.PubkeyToAddress(*pubKey)
-		if targets[addr] || (targets == nil) {
-			m[addr] = p
+		id := p.Node().ID()
+		if targets[id] || (targets == nil) {
+			m[id] = p
 		}
 	}
 	return m
 }
 
+// FindSentryPeers returns a map of the sentry peers
```

Comment:
Move `FindSentryPeers`, `AddValidatorPeer`, `RemoveValidatorPeer`, `GetValidatorPeers`, `GetSentryPeers` to the `istanbul` package.